caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -360,6 +367,10 @@ def train(
             tuning_type = TuningType(tuning_type)
         error.type_check("<FPT65714993E>", TuningType, tuning_type=tuning_type)
 
+        train_stream = train_stream.map(convert_to_generation_record)
```

Comment:
should we add a type_check to make sure `train_stream` is of these 2 types? (same for val stream)

Reply:
handled this in        convert_to_generation_record function instead , if unsupported instance type while reading stream it will error